### PR TITLE
[Tokenizers][TF FE] Fix MUSE conversion

### PR DESCRIPTION
**Pull request by apaniukov**

### CODE REVIEW

**Review by rkazants — COMMENTED** [review]
please productize changes and let us merge it

File: modules/custom_operations/user_ie_extensions/tokenizer/sentence_piece.cpp [hunk 1]
```diff
@@ -157,24 +134,14 @@ bool SentencepieceTokenizer::evaluate(TensorVector& outputs, const TensorVector&
     if(input_element_type == ov::element::string) {
```
Comment (rkazants) [inline modules/custom_operations/user_ie_extensions/tokenizer/sentence_piece.cpp:134]: Please remove artifacts of packed strings: `begin_ids`, etc. above

**Review by rkazants — APPROVED** [review]
one minor comment remained

File: modules/custom_operations/user_ie_extensions/tokenizer/sentence_piece.cpp (hunk 1 above)
Comment (apaniukov) [reply modules/custom_operations/user_ie_extensions/tokenizer/sentence_piece.cpp:129]: Deleted.

File: modules/custom_operations/user_ie_extensions/tokenizer/tensorflow_translators.cpp [hunk 2]
```diff
@@ -197,18 +178,9 @@ ov::OutputVector translate_reshape(const ov::frontend::NodeContext& node) {
...
-        auto reshape = post_translate_string_tensor_output({begins, ends, chars});
-        return {reshape};
-    } else {
-        auto reshape = std::make_shared<Reshape>(tensor, shape, false);
-        return {reshape};
-    }
+    auto reshape = std::make_shared<Reshape>(tensor, shape, false);
+    return {reshape};
+//    }
     // set_node_name(node.get_name(), reshape); // TODO: requires dependencies from TF FE internals
```
Comment (rkazants) [inline modules/custom_operations/user_ie_extensions/tokenizer/tensorflow_translators.cpp:184]: do we need this? Please clean the code a bit.

Comment (rkazants) [inline modules/custom_operations/user_ie_extensions/tokenizer/tensorflow_translators.cpp:184]: I think you should revert these changes, they does not relate MUSE model

Comment (apaniukov) [reply modules/custom_operations/user_ie_extensions/tokenizer/tensorflow_translators.cpp]: Deleted

Comment (apaniukov) [reply modules/custom_operations/user_ie_extensions/tokenizer/tensorflow_translators.cpp]: Reverted.

**Review by rkazants — APPROVED** [review]

**Comment by apaniukov** [comment]
@dtrawins @slyalin sentencepiece operation translated from TF with string input by default. It also allows input of a decomposed string representation, but this must be changed manually after the model is converted. This PR allows you to experiment with data format optimizations without having to recompile the extension.
